src/remoteplayer.h

```diff
@@ -32,6 +32,16 @@ enum RemotePlayerChatResult
 	RPLAYER_CHATRESULT_KICK,
 };
 
+struct CloudSettings
```

Comment:
When i said 'cloudparameters' is fine i meant as the name of the file, not elsewhere. So the file can remain as cloudparameters.h.